src/formatter/builder.ts

```diff
@@ -107,8 +107,15 @@ const FormatterBuilder = {
   },
 
   loadCustomFormatter(customFormatterPath: string, cwd: string) {
-    const fullCustomFormatterPath = path.resolve(cwd, customFormatterPath)
-    const CustomFormatter = require(fullCustomFormatterPath) // eslint-disable-line @typescript-eslint/no-var-requires
+    let CustomFormatter = null
+
+    if (customFormatterPath.startsWith('.')) {
+      const fullCustomFormatterPath = path.resolve(cwd, customFormatterPath)
+      CustomFormatter = require(fullCustomFormatterPath) // eslint-disable-line @typescript-eslint/no-var-requires
+    } else {
+      CustomFormatter = require(customFormatterPath) // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
This should be using `createRequire` to run the require from the cwd

Reply:
Yeah that seems a good point, although per the docs:

> ## module.createRequire(filename)#
> Added in: v12.2.0

Given we're still supporting Node 10 for the time being (its EOL is April 2021), could use https://github.com/sindresorhus/import-cwd instead.

Reply:
I'd suggest using https://yarnpkg.com/package/create-require instead as it will use the node builtins when it can, so `createRequire` on node >= 12.2 `createRequireFromPath` on node >= 10.12, then a polyfil for the rest which is much better

Reply:
Should actually use it for both branches here, which simplifies the logic and has the same end result

Reply:
Added the above, works great for both cases, thanks for pointing it out